specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/Diagnostics.json

```diff
@@ -432,6 +479,212 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/detectorProperties/daprApi/": {
+      "get": {
+        "tags": [
+          "ManagedEnvironments",
+          "Diagnostics"
+        ],
+        "summary": "Get the Dapr components of a Managed Environment.",
+        "description": "Get the Dapr components of a Managed Environment used to host container apps.",
+        "operationId": "ManagedEnvironmentsDiagnostics_ListDaprComponents",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprComponentsCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
```

Comment:
> } [](http://example.com/codeflow?start=7&length=2)

For list operation, you need "x-ms-pageable"

Reply:
Thanks. Updated